bigbluebutton-web/src/java/org/bigbluebutton/api/messaging/converters/messages/CreateMeetingMessage.java

```diff
@@ -1,39 +1,40 @@
 package org.bigbluebutton.api.messaging.converters.messages;
 

```

Comment:
Shouldn't need to change the indentation for the whole file to make the three additions you need.